www/_template/guides/connecting-tools.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layouts/guide.njk
+title: The Snowpack Guide to connecting your favorite tools
+description: 'How do you use your favorite tools in Snowpack? This Guide will help you get started'
+tags: guides
+sidebarTitle: Connecting your favorite tools
```

Comment:
WDYT about having this be “Other Tools” in the Sidebar? That way it’s more of a general guide (as opposed to, say, what if we added a “Sass“ guide would anyone read this guide)?
![Screen Shot 2020-11-11 at 4 12 25 PM](https://user-images.githubusercontent.com/1369770/98875068-b649f200-2438-11eb-90e7-d2ec69b424fd.png)